addons/website/static/src/js/content/menu.js

```diff
@@ -32,7 +32,7 @@ const BaseAnimatedHeader = animations.Animation.extend({
     start: function () {
         this.$main = this.$el.next('main');
         this.isOverlayHeader = !!this.$el.closest('.o_header_overlay, .o_header_overlay_theme').length;
-        this.$dropdowns = this.$el.find('.dropdown, .dropdown-menu');
+        this.$dropdowns = this.$el.find('.dropdown:not(:has(> .o_mega_menu)), .dropdown-menu:not(.o_mega_menu)');
```

Comment:
@younn-o What I meant is that in edit mode, it is possible to scroll the page **then** opening a mega menu (and thus still having the close-on-enter bug with your previous fix). With this change, you change a functionnal behavior which I am not sure make much sense: why do we close dropdown on scroll but not the mega menus :thinking: I am not sure that logic make sense.

Also I tested and it feels strange: depending on the effect you chose for your header scroll, it is nice or ok or seems buggy. With a disappear effect for example, it makes no sense. The mega menu stays open on scroll but the header disappear leaving a floating menu...